test/acceptance/create_webdriver.robot

```diff
@@ -15,38 +15,54 @@ Create Webdriver Creates Functioning WebDriver
 
 Create Webdriver With Bad Driver Name
     [Documentation]    Invalid browser name
-    Run Keyword And Expect Error    'Fireox' is not a valid WebDriver name
-    ...    Create Webdriver    Fireox
+    Run Keyword And Expect Error    'Fireox' is not a valid WebDriver name    Create Webdriver
+    ...    Fireox
 
 Create Webdriver With Duplicate Arguments
     [Documentation]    Invalid values in arguments
     ${kwargs}=    Create Dictionary    arg=1
-    Run Keyword And Expect Error    Got multiple values for argument 'arg'.
-    ...    Create Webdriver    Firefox    kwargs=${kwargs}    arg=2
+    Run Keyword And Expect Error    Got multiple values for argument 'arg'.    Create Webdriver
+    ...    Firefox    kwargs=${kwargs}    arg=2
 
 Create Webdriver With Bad Keyword Argument Dictionary
     [Documentation]    Invalid arguments types
-    Run Keyword And Expect Error    kwargs must be a dictionary.
-    ...    Create Webdriver    Firefox    kwargs={'spam': 'eggs'}
+    Run Keyword And Expect Error    kwargs must be a dictionary.    Create Webdriver    Firefox
+    ...    kwargs={'spam': 'eggs'}
+
+Get Browser Info
```

Comment:
I agree with @aaltat that the test below isn't testing much. I'm not sure how much could actually be verified, because knowing exactly what information is returned may be hard. Perhaps the test should just access all information we think should always be there.

Regardless what is done, the last lines that ignore all errors should just be removed. They just add noise.